text/accumulation.tex

```diff
@@ -30,118 +34,206 @@ \subsection{Preimage Integration}
+Next, the queue-editing function $E$, which is essentially a mutator function for items such as those of $\ready$, parameterized by a dictionary similar to the items in $\accumulated$. It is used to update queues of work-reports when some of them are accumulated. Functionally, it removes all entries whose work-report's hash is in the parameter's keys, removes any dependencies which appear in said set and, crucially, removes any entries whose segment-root correspondences diverge from those in the dictionary. Formally:
+\begin{equation}
+  E\colon\left\{\begin{aligned}
+      &(\seq{(\mathbb{W}, \{\H\})}, \dict{\H}{\H}) \to \seq{(\mathbb{W}, \{\H\})} \\
+    &(\mathbf{r}, \mathbf{x}) \mapsto \left[
+      (w, \mathbf{d} \setminus \keys{\mathbf{x}})
+      \,\middle\vert\,
+      \left\{\,\begin{aligned}
+        &(w, \mathbf{d}) \orderedin \mathbf{r} ,\\
+        &\mathcal{H}(\se(w)) \not\in \keys{\mathbf{x}},\\
+        &\mathbf{x} \cup w_\wr¬srlookup = w_\wr¬srlookup \cup \mathbf{x}
+      \end{aligned}\right.
+      \right]
+  \end{aligned}\right.
+\end{equation}
 
-We first denote the set of deferred transfers as $\mathbb{T}$, noting that a transfer includes a memo component $m$ of $\mathcal{M} = 128$ octets, together with the service index of the sender $s$, the service index of the receiver $d$, the amount of tokens to be transferred $a$ and the gas limit $g$ for the transfer. Formally:
-\begin{align}
-  \mathbb{T} \equiv \ltuple\isa{s}{\N_S}\ts\isa{d}{\N_S}\ts\isa{a}{\N_B}\ts\isa{m}{\Y_{\mathsf{M}}}\ts\isa{g}{\N_G}\rtuple
-\end{align}
+We further define the accumulation priority queue function $Q$, which provides the sequence of work-reports which are accumulatable given the work-reports which have hitherto been accumulated and a set of not-yet-accumulated work-reports and their dependencies.
+\begin{equation}
+  Q\colon\left\{\begin{aligned}
+    &(\seq{(\mathbb{W}, \{\H\})}, \dict{\H}{\H}) \to \seq{\mathbb{W}} \\
+    &(\mathbf{r}, \mathbf{a}) \mapsto \begin{cases}
+      [] &\when w = \none \\
+      \mathbf{g} \frown Q(E(\mathbf{r}, \mathbf{g}), \mathbf{a} \cup \srmap(\mathbf{g}))\!\!\!\! &\otherwise \\
+      \multicolumn{2}{l}{\quad\where \mathbf{g} = [w \mid (w, \{\}) \orderedin \mathbf{r}]}
+    \end{cases}\!\!\!\!
+  \end{aligned}\right.
+\end{equation}
+
+We may now define the sequence of accumulatable work-reports in this block as $\mathbf{W}^*$:
+\begin{equation}
+  \mathbf{W}^* \equiv \mathbf{W}^! \frown Q(\wideparen{\ready_{m\dots}} \concat \wideparen{\ready_{\dots m}} \concat \mathbf{W}^Q, \wideparen{\accumulated})
```

Comment:
I assume m is the current slot mod the epoch period, would be good to clarify this here?